packages/expect/src/utils.ts

```diff
@@ -100,7 +102,7 @@ export const getObjectSubset = (
   subset: any,
   seenReferences: WeakMap<object, boolean> = new WeakMap(),
 ): any => {
-  /* eslint-enable */
+  /* eslint-enable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
I'd prefer a full `enable` followed by a new `disable` - that way we don't inadvertently forget to enable some other lint rule

Reply:
That's what the `eslint-comments` plugin catches, so that shouldn't be possible.

Can still try and make it work if you'd like :)

Reply:
ah, didn't know `disable-enable-pair` picked up on this. Sweet! Can leave it alone, then 👍